VirusTotal/dist/presence.js

```diff
@@ -0,0 +1,229 @@
+    presenceData.details = "Scan url > " + scanurl;
+    presenceData.state = results;
+    presence.setActivity(presenceData);
+  
+   }else if (document.location.pathname == "/gui/sign-in"){  
+    presenceData.details = "Log in to account...";
+    presence.setActivity(presenceData);
+
+   }  else if (document.location.pathname.includes("/gui/user/")){
+    nameuser = document.querySelector("body > vt-virustotal-app").shadowRoot.querySelector("#authChecker > user-view").shadowRoot.querySelector("#pageWrapper > div.wrapper > vt-ui-generic-card > div:nth-child(2) > div.avatar-name > div > h3").innerText;    
+    presenceData.details = "Browsing though a profile";
+    presenceData.state = nameuser + "'s profile";
+    presence.setActivity(presenceData);
+  
+   }else if (document.location.pathname.includes("/gui/home/search")){
+    searchtext = document.querySelector("body > vt-virustotal-app").shadowRoot.querySelector("#authChecker > home-view").shadowRoot.querySelector("#searchbar").shadowRoot.querySelector("#searchInput").shadowRoot.querySelector("#input").value;    
+    presenceData.details = "Searching...";
+    presenceData.state = searchtext;
+    presence.setActivity(presenceData);
+  
+   }else if (document.location.pathname.includes("/gui/settings")){
+    presenceData.details = "Change account settings...";
+    presence.setActivity(presenceData);
+  
+   }else if (document.location.pathname.includes("/gui/home/upload")){
+    presenceData.details = "Upload a file...";
+    presence.setActivity(presenceData);
+  
+   }else if (document.location.pathname == "/gui/join-us"){
+    username3 = document.querySelector("body > vt-virustotal-app").shadowRoot.querySelector("#authChecker > join-us-view").shadowRoot.querySelector("#username").shadowRoot.querySelector("#input").value;
+    presenceData.details = "Create a account...";
+    presenceData.state = "Maybe known as " + username3;
+    presence.setActivity(presenceData);
+
+   }else if (document.location.pathname.includes("/gui/home/url")){
+    presenceData.details = "Search for a URL...";
+    presenceData.state = "Think about typing.";
+    //presence.setActivity(presenceData);
+    url2 = document.querySelector("body > vt-virustotal-app").shadowRoot.querySelector("#authChecker > home-view").shadowRoot.querySelector("#urlSearchInput").shadowRoot.querySelector("#input").value;
+    if (url2 => 1){
+        presenceData.details = "Search for a URL...";
+        presenceData.state = url2;
+       // presence.setActivity(presenceData);
+    }
+    presence.setActivity(presenceData);
+   }
+       
+
+ 
+   
+        //presenceData.state = "Lurking on LautFM"; break;
+        //default : presenceData.state = "ZZzzzZZ"; 
+        ////
+        //playingnow = document.querySelector("#app > div.fixed.fixed--top > div > a > div > div > span > b").innerText;
+        //presenceData.details = "Playing " + playingnow;
```

Comment:
Why is this in here?

Reply:
It is a template if i forgot anything :D
should I remove this?

Reply:
yea try to keep it as clean as possible, but you can leave notes etc. just not a whole template